src/MiNET/MiNET/Net/Transaction.cs

```diff
@@ -811,14 +845,25 @@ internal static InventoryMismatchTransaction ReadData(Packet packet)
 	public class ItemUseTransaction : Transaction
 	{
 		public McpeInventoryTransaction.ItemUseAction ActionType { get; set; }
+
+		public TriggerType TriggerType { get; set; }
+
 		public BlockCoordinates Position { get; set; }
+
 		public int Face { get; set; }
+
 		public int Slot { get; set; }
+
 		public Item Item { get; set; }
+
 		public Vector3 FromPosition { get; set; }
+
 		public Vector3 ClickPosition { get; set; }
+
 		public uint BlockRuntimeId { get; set; }
 
+		public PredictedResult ClientInteractPrediction { get; set; }
```

Comment:
interesting thing that may help us to validate user actions. Check using in PM if it exists there